libcontainer/cgroups/cgroups.go

```diff
@@ -30,7 +30,7 @@ type Manager interface {
 	Path(string) string
 
 	// Sets the cgroup as configured.
-	Set(container *configs.Config) error
+	Set(r *configs.Resources) error
```

Comment:
Added

Reply:
The comment still doesn't mention that `Set(nil)` after `Set(r)` will use the (now cached) new r rather than the one specified during Manager creation (in fact it says the opposite).

Reply:
You're quite right -- I have now fixed this, as well as the one-before-last commit description to correctly document/explain what is going on.

Reply:
Maybe I'm missing something, but GitHub is still showing me this as the diff for the `Set` doc comment:

```
	// Set sets cgroup resources parameters/limits. If the argument is nil,
	// the resources specified during Manager creation are used.
```

Which is the same as it was before?

Reply:
fixed now (guess I forgot to amend the last commit)